src/main/java/edu/oakland/production/database/DatabaseGisManagerImplementation.java

```diff
@@ -0,0 +1,33 @@
+package edu.oakland.production.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+
+public class DatabaseGisManagerImplementation {
```

Comment:
This needs to implement the `DatabaseGisManager` Java interface.
```suggestion
public class DatabaseGisManagerImplementation implements DatabaseGisManager {
```